bokehjs/src/lib/models/layouts/tabs.ts

```diff
@@ -112,7 +112,7 @@ export class TabsView extends LayoutDOMView {
           this.change_active(i)
       })
       const tooltip = this.tooltips[i]
-      if (tooltip !== null) {
+      if (tooltip !== null && tooltip !== undefined) {
```

Comment:
Use the following instead:
```suggestion
      if (tooltip != null) {
```
Generally avoid `===` and `!==` unless you want to distinguish between `null` and `undefined`.